### ISSUE TITLE
Can a list of training and test sets be provided?

### ISSUE THREAD

**Issue by molly00ecla** [rcuocolo/PROSTATEx_masks#17]
Hi, can you provide the image list of the training and test sets ? 
In the paper(https://pubmed.ncbi.nlm.nih.gov/33634932/), it was described that  "In detail, 99 cases from the PROSTATEx dataset were used as the training set in a 5-fold cross-validation fashion and the remaining 105 patients as the test set. The latter includes patients who have undergone biopsy for suspect of csPCa." However, I checked the file "PROSTATEx_Classes.csv" and found there were 99 cases with biopsy information, while another 101 cases without biopsy information. It would be appreciated if you could provide a list of test sets.

**Comment by rcuocolo** [comment]
The segmentation model data employed all exams (T2 images), even those whose lesions were not included in the lesion mask dataset due to issues found during the auditing process. All cases with biopsy (i.e., included in the PROSTATEx2 dataset) were used for training and validation, while the remaining cases (i.e., without biopsy information) were used for testing. The images and masks used are those included in the repository.
